Formula/chart-testing.rb

```diff
@@ -0,0 +1,35 @@
+class ChartTesting < Formula
+  desc "Testing and linting Helm charts"
+  homepage "https://github.com/helm/chart-testing"
+  url "https://github.com/helm/chart-testing.git",
+      {
+        tag:      "v3.0.0",
+        revision: "50db473a1e68c605b18d82f019d83ea401542213",
+      }
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "helm" => :test
+  depends_on "yamllint" => :yamllint
```

Comment:
What `:yamlint` means here?

Reply:
This was meant to be `:test`. I changed it now.